android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -843,6 +844,46 @@ sealed class ContentAction : BrowserAction() {
  * [BrowserAction] implementations related to translating a web content page.
  */
 sealed class TranslationsAction : BrowserAction() {
+    /**
+     * Indicates that the translations toolkit expects the user may want to translate the page on
```

Comment:
nit: Maybe we should omit the uses of `toolkit` as this refers back to the Gecko specific implementation, browser state is engine agnostic.

Reply:
Thanks, updated language from `toolkit` to `translations engine` and avoided Gecko specific language!